src/garage/torch/policies/gaussian_mlp_policy.py

```diff
@@ -115,6 +115,22 @@ def get_action(self, observation):
                     dict(mean=dist.mean.squeeze(0).numpy(),
                          log_std=(dist.variance**.5).log().squeeze(0).numpy()))
 
+    def forward(self, *obervations):
+        """Forward method.
+
+        Args:
+            obervations (torch.Tensor): Observation from the environment.
+                Shape is :math:`env_spec.observation_space`.
+
+        Returns:
+            torch.distributions.independent.Independent: Independent
```

Comment:
@avnishn can you please explain why this is returning `Independent` instead of just a batch of samples?

also, we still need shape information. AFAICT this type still has a shape.

saying it returns `Independent` is at best unhelpful and at worst misleading.

Reply:
sure thing!

Reply:
To answer your first question, users need access to the underlying distribution in some situations (I'm thinking of some situations when I wrote SAC). They could however be rewritten to return a batch of randomly sampled actions. However, I think that that would be out of the scope of this PR.

You are however correct to say that this should have a shape.

Reply:
However why was there a need to add a forward method to this function? @yonghyuc